### ISSUE TITLE
Client: Add tee freeze state

### ISSUE THREAD

**Issue by sepulzera** [teeworlds/teeworlds#2173]
I would like to see support for freeze. That is for example used in DDRace and in zCatch (Anticamper). The tee can not move meanwhile.

Currently they use the ninja skin for that.

**Comment by AssassinTee** [comment]
Can you give an expample how you would do that? I mean use another skin or with some kind of animation

**Comment by Dune-jr** [comment]
This is not used by any official gametype or would-be. If we start throwing in client features to support specific mods we will never see the end of it.

**Comment by Sonix-** [comment]
> This is not used by any official gametype or would-be. If we start throwing in client features to support specific mods we will never see the end of it.


I agree with Dune

**Comment by sepulzera** [comment]
Still yet we couldn't decide on a way to distribute content from inofficial servers. But why don't support certain more used mechanics in mods? Adding a freeze skin shouldn't cause too much trouble but would help to mitigate the re-usage of the vanilla ninja skin.

And if there are 3 or 4 additional skins provided for mods only, I would be fine, too. I just don't see any real drawback here.

**Comment by oy** [comment]
Yeah, adding a special freeze skin that renders your're tee within an ice cube might be an option?

**Comment by Dune-jr** [comment]
> Yeah, adding a special freeze skin that renders your're tee within an ice cube might be an option?

It would need to have some special flags so that it could not be used by players (like x_ninja), but could be set by the server.

**Comment by fokkonaut** [comment]
I dont think this is worth it, most mod devs would probably stick to the good old ninja skin with damage indicator. I see no real reason adding this.

**Comment by oy** [comment]
It could be used when the game is paused. A visual indicator - rendering all players frozen.

**Comment by AssassinTee** [comment]
> 
> 
> It could be used when the game is paused. A visual indicator - rendering all players frozen.

so we could add "frozen projectiles", too

**Comment by Ryozuki** [comment]
a freeze gun too
